README.md

```diff
@@ -1,92 +1,137 @@
+|                                [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/KateAnn19)                                |                            [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/sfritz24)                             |                          [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/samkester)                           |                          [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/arn-foto)                           |
+|                [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/kate-mcgee/)                |                 [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/shanon-fritz/)                 |                [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/sam-kester/)                |                 [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/anthonyrnavarro/)                 |
 
 <br>
-<br>
-
-🚫 Optional examples of using images with links for your tech stack, make sure to change these to fit your project
-
-![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg)
-![React](https://img.shields.io/badge/react-v16.7.0--alpha.2-blue.svg)
-![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)
-
-🚫 more info on using badges [here](https://github.com/badges/shields)
-
-- Fork and clone the repo to install it as your own remote.
-  - **note** please [be sure to set your remote](https://help.github.jp/enterprise/2.11/user/articles/changing-a-remote-s-url/) for this repo to point to your Labs Team Front End Repository.
-- run: `npm install` to download all dependencies.
-- run: `npm start` to start your local development server.
-
-> When using Okta for authentication, the app will need to run locally on port 3000.
-
-### Key Features
-
-- feature one
-- feature two
-- feature three
-- feature four
-- feature five
-
-#### Front end deployed to `🚫insert service here`
-
-#### [Back end](🚫link to back end repo here) built using:
 
-#### 🚫 back end framework goes here
+## Key Features
 
-- point one
-- point two
-- point three
-
-🚫 List the rest of the back end end features and libraries in the same format as the framework above
+- Users and organizations can create and update public-facing profiles.
+- Users can apply to join organizations as a microentrepreneur.
+- Users can apply to create a new partner organization.
+- Admins for an organization can accept or reject applications.
 
 # APIs
 
```

Comment:
This endpoints are very clear and easy to understand, awesome!

Reply:
Great job!